examples/MyApp/config/yard_api.yml

```diff
@@ -1,5 +1,9 @@
 centered: true
+readme: doc/index.md
 static:
+  - title: Secret Page
+    path: doc/secret.md
```

Comment:
i would probably not advocate throwing secret pages out there in the wild that merely lack a sidebar link xD